domain/src/main/java/org/oppia/android/domain/hintsandsolution/HintHandlerProdImpl.kt

```diff
@@ -301,7 +310,8 @@ class HintHandlerImpl private constructor(
           latestAvailableHintIndex = nextHelpIndexToShow.nextAvailableHintIndex
         }
         SHOW_SOLUTION -> solutionIsAvailable = true
-        else -> {} // Nothing else to do.
+        else -> {
+        } // Nothing else to do.
```

Comment:
Why this change?

Reply:
Well, this is the second instance in this PR that there is a change that I didn't commit (at least not knowingly). I think this was changed when I formatted the code using Ctrl + Alt + L in Android Studio. Anyways, I have reverted this change.